astropy/io/fits/scripts/fitsheader.py

```diff
@@ -401,7 +401,10 @@ def main(args=None):
                      'Optional arguments allow the desired extension(s), '
                      'keyword(s), and output format to be specified. '
                      'Note that in the case of a compressed image, '
-                     'the decompressed header is shown by default.'))
+                     'the decompressed header is shown by default. \n\n'
+                     f'This script is part of the Astropy package, version {__version__}. '
+                     'See https://docs.astropy.org/en/latest/io/fits/usage/scripts.html#module-astropy.io.fits.scripts.fitsheader for further documentation.'),
+                     formatter_class=argparse.RawDescriptionHelpFormatter)
```

Comment:
Would be great if we can improve consistency between the scripts, so please:
- Move the description in a global variable as fitscheck and fitsdiff,
- Use a multi-line string (`""" ... """`)
- Always put this message at the end of the description and split long lines (e.g. after the URL).

Thanks!

Reply:
Thanks for the feedback, should be fixed in commit https://github.com/astropy/astropy/pull/11144/commits/a915758e804fecaf1d884d24ee639b8c7f028d30